Cargo.lock

```diff
@@ -156,9 +156,9 @@ checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
 
 [[package]]
 name = "bitflags"
-version = "2.4.0"
```

Comment:
Sorry I didn't understand the commit description  ```
* Keep rustix updated and avoid a dependency mismatch with
   parsec-tool.
```
Will the bitflags crate have different versions in parsec and parsec-tool if we update rustix?

Reply:
https://github.com/tgonzalezorlandoarm/parsec-tool/actions/runs/6972139517/job/18973702435

When checking the current mismatches between parsec and parsec-tool:
```
---------------------mistmatches----------------------


bitflags                  ['v2.4.1', 'v2.4.0']
rustix                    ['v0.38.18', 'v0.38.21']
```
As dependabot pointed out, rustix can be updated to its latest. So can bitflags. This PR is doing that.
This is done in parsec-tool as well as part of this https://github.com/parallaxsecond/parsec-tool/pull/116

Reply:
This will avoid a dependency mismatch between parsec and parsec-tool